### DISCUSSION TITLE
Create datagrid table on form

### DISCUSSION THREAD

**Discussion by hanafiadib** [abydahana/aksara#446] [Q&A]
how to create a form that has datatable to support parent-child table structure?

**Comment by abydahana** [comment]
Did you mean the table to handle a field relation?

**Reply by hanafiadib** [reply]
yes, form with datatable for detail/child relation

**Comment by FlavioSuarez** [comment]
I'd like to know too!!
A form with:
- the fields of a parent file and 
-  a datagrid with columns of each of the child files

And what about a form with too many fields? Can they be grouped on tabs?

TIA!
